docs/configuring-metabase/appearance.md

```diff
@@ -49,6 +49,14 @@ You can customize the colors that Metabase uses throughout the app:
 
 You can choose up to 24 hex values. If you choose fewer than 24 colors, Metabase will auto-generate colors to fill in the rest of the values.
 
+Custom colors are available for the following charts:
```

Comment:
Is this right? What about conditional formatting for tables? Maps? Donuts?

Reply:
@likeshumidity could you confirm please? :) 

I took the information from this ticket: https://metabase.zendesk.com/agent/tickets/15299

Reply:
Coming up with a longer list actually, it seems that all of the following include the custom colors as a selector:
- Line
- Bar
- Combo
- Area
- Row
- Waterfall
- Scatter
- Pie
- Trend
- Progress
- Gauge

And the below do not:
- Number
- Trend
- Funnel
- Table - Conditional Formatting
- Pivot Table - Conditional Formatting
- Map

So it may make more sense to callout the ones that don't instead?